### PR TITLE
Clarify keep_input_roots docs

### LINKED ISSUE #1151: Clarify behaviour of simplify with keep_input_roots=true and unary nodes
The documentation for keep_input_roots is misleading in the presence of unary nodes. We need to add a clarification like

> if keep_unary is True the path of unary nodes and edges from the MRCA to the root will also be retained.

### PR DESCRIPTION
**Pull request by hyanwong**
Fixes #1151, with @petrelharp 's rewording (I changed "root" to MRCA though, which I think was a simple typo in his suggestion)

### CODE REVIEW

**Review by jeromekelleher — APPROVED** [review]
LGTM

File: python/tskit/tables.py [hunk 1]
```diff
@@ -2541,9 +2541,10 @@ def simplify(
         :param bool keep_unary: If True, any unary nodes (i.e. nodes with exactly
             one child) that exist on the path from samples to root will be preserved
             in the output. (Default: False)
-        :param bool keep_input_roots: If True, insert edges from the MRCAs of the
-            samples to the roots in the input trees. If False, no topology older
-            than the MRCAs of the samples will be included. (Default: False)
+        :param bool keep_input_roots: Whether to retain history ancestral to the
+            MRCA of the samples. If False, no topology older than the MRCAs of the
+            samples will be included; the roots of all trees in the returned tree
+            sequence will be MRCAs in the original tree sequence. (Default: False)
```
Comment (petrelharp) [inline python/tskit/tables.py:2547]: this MRCA should be "root", I think.

Comment (hyanwong) [reply python/tskit/tables.py:2547]: Oh, did I miscorrect you, sorry. You said "the roots of all trees will be roots in the original tree sequence", which I find difficult to parse, and I think (?) is wrong isn't it? If `False`, the roots in the returned TS will be the MRCAs in the original TS. If `True`, the roots in the returned TS will be the same roots as in the original TS. I think, anyway?

Comment (hyanwong) [reply python/tskit/tables.py:2547]: Do you mean, perhaps

> Whether to retain history ancestral to the
> MRCA of the samples. If ``False``, no topology older than the MRCAs of the
> samples will be included. If ``True`` the roots of all trees in the returned tree
> sequence will be the same roots as in the original tree sequence. (Default: False)

Comment (petrelharp) [reply python/tskit/tables.py:2547]: Ah, yes, sorry - your suggestion is in fact exactly what I was thinking.  I suppose that the current statement is not wrong, but a bit confusing.

Comment (hyanwong) [reply python/tskit/tables.py:2547]: I'll change it to the quote above "Whether to retain history ancestral to the
MRCA of the samples. If False, no topology older than the MRCAs of the
samples will be included. If True the roots of all trees in the returned tree
sequence will be the same roots as in the original tree sequence. (Default: False)"

Comment (hyanwong) [reply python/tskit/tables.py]: Done

**Review by petrelharp — APPROVED** [review]

**Pull request merged**